lib/scryptenc/scryptenc_cpuperf.c

```diff
@@ -26,97 +26,24 @@
  * This file was originally written by Colin Percival as part of the Tarsnap
  * online backup system.
  */
-#include "bsdtar_platform.h"
-
 #include <sys/time.h>
 
 #include <stdint.h>
 #include <stdio.h>
-#include <time.h>
 
 #include "crypto_scrypt.h"
+#include "monoclock.h"
```

Comment:
You might possibly want to wait until the scrypt code has been tested a bit more before importing the `scryptenc_cpuperf` monoclock change.